src/backend/utils/logger.js

```diff
@@ -32,7 +33,18 @@ if (process.env.LOG_FILE) {
   destination = pino.destination(process.stdout.fd);
 }
 
-const logger = pino(options, destination);
+const streamToElastic = pinoElastic({
+  index: (logTime) => {
+    // the logTime is a ISO 8601 formatted string of the log line
+    return `Telescope-${logTime.substring(5, 10)}`;
+  },
+  consistency: 'one',
+  node: `${process.env.ELASTIC_URL}:${process.env.ELASTIC_PORT}`,
+  'es-version': 7,
+  'flush-bytes': 1000,
```

Comment:
I think it's better to store this number in a constant and use it here (avoid [magic numbers](https://codeburst.io/software-anti-patterns-magic-numbers-7bc484f40544))